src/views/ModelPlan/TaskList/Beneficiaries/Frequency/index.tsx

```diff
@@ -362,28 +368,53 @@ const Frequency = () => {
                         scrollElement="precedenceRules"
                         error={!!flatErrors.precedenceRules}
                       >
-                        <Label
-                          htmlFor="beneficiaries-precedence-rules"
-                          className="maxw-none"
-                        >
+                        <Label htmlFor="precedenceRules">
                           {beneficiariesT('precedenceRules.label')}
                         </Label>
 
-                        <p className="text-base margin-0 line-height-body-3">
+                        <p className="text-base margin-top-1 margin-bottom-0 line-height-body-3">
                           {beneficiariesT('precedenceRules.sublabel')}
                         </p>
 
                         <FieldErrorMsg>
                           {flatErrors.precedenceRules}
                         </FieldErrorMsg>
 
-                        <Field
-                          as={TextAreaField}
-                          className="height-15"
-                          error={flatErrors.precedenceRules}
-                          id="beneficiaries-precedence-rules"
-                          data-testid="beneficiaries-precedence-rules"
-                          name="precedenceRules"
+                        {getKeys(beneficiaryPrecedenceConfig.options).map(
+                          key => (
+                            <Fragment key={key}>
+                              <Field
+                                as={CheckboxField}
+                                id={`beneficiaries-precedence-rules-${key}`}
+                                data-testid={`beneficiaries-precedence-rules-${key}`}
+                                name="precedenceRules"
+                                label={beneficiaryPrecedenceConfig.options[key]}
+                                value={key}
+                                checked={values.precedenceRules.includes(key)}
+                              />
+
+                              {values.precedenceRules?.includes(key) && (
+                                <div className="margin-left-4">
+                                  <span>
+                                    {beneficiariesT('precedenceRulesYes.label')}
+                                  </span>
+                                  <Field
+                                    as={TextAreaField}
+                                    className="height-15"
+                                    error={flatErrors.precedenceRules}
```

Comment:
This error isn't really needed, and honestly we overuse them throughout the app because we don't have much validation to even trigger errors.  If we did have validation here the error would need to be converted to a boolean - something like `!!flatErrors.precedenceRules`